src/solaredge_modbus/__init__.py

```diff
@@ -629,6 +661,30 @@ def __init__(self, offset=False, *args, **kwargs):
             "energy_reactive_scale": (0x9d64 + self.offset, 1, registerType.HOLDING, registerDataType.SCALE, int, "Energy (Reactive) Scale Factor", "", 3)
         }
 
+class StorEdge(SolarEdge):
+    
+    def __init__(self, *args, **kwargs):
+        self.model = "StorEdge"
+        self.wordorder = Endian.LITTLE
+
+        super().__init__(*args, **kwargs)
+
+        self.registers = {
+            "export_control_mode": (0xe000, 1, registerType.HOLDING, registerDataType.UINT16, int, "Export Control Mode", EXPORT_CONTROL_MODE_MAP, 1),
+            "export_control_limit_mode": (0xe001, 1, registerType.HOLDING, registerDataType.UINT16, int, "Export Control Limit Mode", EXPORT_CONTROL_LIMIT_MAP, 1),
+            "export_control_site_limit": (0xe002, 2, registerType.HOLDING, registerDataType.UINT32, int, "Export Control Site Limit", "", 1),
+            
+            "storedge_control_mode": (0xe004, 1, registerType.HOLDING, registerDataType.UINT16, int, "StorEdge Control Mode", STOREDGE_CONTROL_MODE, 1),
+            "storedge_ac_charge_policy": (0xe005, 1, registerType.HOLDING, registerDataType.UINT16, int, "StorEdge AC Charge Policy", STOREDGE_AC_CHARGE_POLICY, 1),
+            "storedge_ac_charge_limit": (0xe006, 2, registerType.HOLDING, registerDataType.UINT32, int, "StorEdge AC Charge Limit (kWh or %)", "", 1),
+            "storedge_backup_reserved": (0xe008, 2, registerType.HOLDING, registerDataType.UINT32, int, "StorEdge Backup Reserved (%)", "", 1),
+            "storedge_default_charge_mode": (0xe00a, 1, registerType.HOLDING, registerDataType.UINT16, int, "StorEdge Default Charge Mode", STOREDGE_CHARGE_DISCHARGE_MODE, 1),
+
+            "storedge_remote_command_timeout": (0xe00b, 2, registerType.HOLDING, registerDataType.UINT32, int, "StorEdge Remote Command Timeout", "", 1),
+            "storedge_remote_command_mode": (0xe00d, 1, registerType.HOLDING, registerDataType.UINT16, int, "StorEdge Remote Command Mode", STOREDGE_CHARGE_DISCHARGE_MODE, 1),
+            "storedge_remote_charge_limit": (0xe00e, 2, registerType.HOLDING, registerDataType.UINT32, int, "StorEdge Remote Command Charge Limit", "", 1),
+            "storedge_remote_discharge_limit": (0xe010, 2, registerType.HOLDING, registerDataType.UINT32, int, "StorEdge Remote Command Discharge Limit", "", 1),
```

Comment:
Wrong data type. e006, e008, f70e and f710 are Float32 and that's most likely the reason why remote command does not work.

Reply:
fixed